model/inline/src/main/scala/aqua/model/inline/raw/CollectionRawInliner.scala

```diff
@@ -15,22 +16,9 @@ object CollectionRawInliner extends RawInliner[CollectionRaw] {
   override def apply[S: Mangler: Exports: Arrows](
     raw: CollectionRaw,
     propertiesAllowed: Boolean
-  ): State[S, (ValueModel, Inline)] = unfoldCollection(raw)
-
-  def unfoldCollection[S: Mangler: Exports: Arrows](
-    raw: CollectionRaw,
-    assignToName: Option[String] = None
-  ): State[S, (ValueModel, Inline)] =
+  ): State[S, (ValueModel, Inline)] = 
     for {
       streamName <- raw.collectionType match {
-        case _: StreamType =>
-          assignToName.fold(
-            Mangler[S].findAndForbidName("stream-inline")
-          )(State.pure)
-        case _: StreamMapType =>
-          assignToName.fold(
-            Mangler[S].findAndForbidName("stream_map-inline")
-          )(State.pure)
         case _: CanonStreamType => Mangler[S].findAndForbidName("canon_stream-inline")
```

Comment:
Seems like this case is impossible? We create `CollectionRaw` with `ArrayType` or `OptionType`. Or am I wrong?

Reply:
this is for scala compiler. Or we can create one more type only for option and array...

Reply:
Lets leave a comment for now